config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Users routes
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show', as: 'user'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  # Posts routes
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show', as: 'post'
```

Comment:
- [OPTIONAL] Hey! You can simplify your routes code like this:

```ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

By using the `resources` method with the appropriate options, you can achieve the same routing behavior more concisely. This helps keep your codebase clean and easy to manage. Keep up the good work! 🚀👨‍💻